File: C++/Modulo.cpp
#include <iostream>
#include <vector>

int main()
{

    int value = 42;

    std::vector<int> input;
    std::vector<int> remainder;

    for(int i = 0 ; i < 10 ; i++)
    {
        int input_temp;
        std::cin >> input_temp;
        if(input_temp <= 1000)
        {
            input.push_back(input_temp);
        }
        else return 0;
    }

    for(int i = 0 ; i< 10 ; i++)
    {
        int add_flag = 0;
        for(int k = 0 ; k< remainder.size() ; k++)
        {
            if(remainder[k] == input[i]%value)
            {
                add_flag++;
                break;
            }
        }
        if(add_flag == 0)
        {
            remainder.push_back(input[i] %value);
        }
    }

    std::cout << remainder.size();

    return 0;
}

File: Java/Modulo.java
import java.lang.Math;
import java.util.ArrayList;

public class Modulo {
    public static void main(String[] args) {

        ArrayList<Integer> input = new ArrayList<Integer>();
        ArrayList<Integer> remainder = new ArrayList<Integer>();
        int value = 3;

        for (int i = 0; i < 10; i++) {
            int temp = (int) Math.round(Math.random() * 100);
            input.add(temp);
        }

        for (int i = 0; i < 10; i++) {

            /* Checking the array if there're exist the same */ 
            boolean add_flag = false;
            for(int k : remainder)
            {
                if( k == input.get(i) % value)
                {
                    add_flag = true;
                    break;
                }

            }
            if(!add_flag)
            {
                remainder.add(input.get(i) % value);
            }
        }

        System.out.println(remainder.size());

    }
}
